generators/kubernetes-knative/__snapshots__/knative.spec.ts.snap

```diff
@@ -27,7 +27,7 @@ exports[`generator - Knative Using Helm generator type MySQL and PostgreSQL micr
 
 ## Preparation
 
-- Knative depends on Istio. In order to use the manifests generated by k8s-knative generator, you should have istio and kntaive installed
+- Knative depends on Istio. In order to use the manifests generated by k8s-knative generator, you should have istio and knative installed
```

Comment:
It seems like Istio and Knative should both be capitalized because they're nouns, not commands.

Reply:
updated 😉